### PR TITLE
style:change basic ui

### PR DESCRIPTION
**Pull request by debarghya472**

### CODE REVIEW

**Comment by s-ayush2903** [comment]
@debarghya472 Please consider attaching the visuals of the changes that are proposed in this PR and it would be somewhat better if you follow some semantic PR titles and commit messages that reflect the changes made. And also provide some relevant description as well
* For Example:- You can rephrase this PR's title as:
style: Basic UI changes 
In the same way commit message can also be rephrased

Following these will make it easier for all to make track of changes that are made in the merged patches

**Comment by debarghya472** [comment]
Screenshot 2020-06-19 at 9 43 56 PM

Screenshot 2020-06-21 at 12 31 15 AM



Requesting suggestion about UI

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
-    <color name="event_card_background">#55738485</color>
+    <color name="event_card_background">#80003153</color>
 
     <color name="white">#ffffff</color>
     <color name="black">#000000</color>
     <color name="transparent">#00000000</color>
 
-    <color name="navigation_background">#aa646B6D</color>
-    <color name="event_detail_card_background">#661E263F</color>
+    <color name="navigation_background">#99000000</color>
```
Comment (s-ayush2903) [inline app/src/main/res/values/colors.xml]: Can you please add some relevant name to the color, something like that lets one know what color it is, similar to the naming practices that have been followed in the above lines of the same file

**Review by s-ayush2903 — CHANGES_REQUESTED** [review]
As the `HomeFragment.java` class has been deleted, please explain where its corresponding xml layout is being inflated.
* In simpler words, please explain why homeFragment has been deleted and what is the significance of its corresponding layout even after the deletion of corresponding host class

**Comment by s-ayush2903** [comment]
* As I can see in the visual attached, there exists no text below third circular icon(that kinda looks like image/gallery option), please take a note of it. 
* Why there is a bottom navigation bar on the screen(this may be due to emulator), test these changes on some real device and update the screenshot here

**Review by amangrobo — CHANGES_REQUESTED** [review]
Mention the reason for deleting the HomeFragment

**Comment by debarghya472** [comment]
- In an attempt to delete the unnecessary code i've mistakenly deleted the HomeFragment.java file. I've re-added it back

- Secondly I've attached a vertical scrollView with the buttons so the rest are not visible

**Comment by debarghya472** [comment]
@amangrobo requesting review

**Review by amangrobo — APPROVED** [review]

**Pull request merged**